ui/page/transaction/transaction_details_page.go

```diff
@@ -90,7 +90,7 @@ type TxDetailsPage struct {
 	moreOptionIsOpen bool
 }
 
-func NewTransactionDetailsPage(l *load.Load, transaction *sharedW.Transaction, _ /*isTicket*/ bool) *TxDetailsPage {
+func NewTransactionDetailsPage(l *load.Load, wallet sharedW.Asset, transaction *sharedW.Transaction, _ /*isTicket*/ bool) *TxDetailsPage {
```

Comment:
The selected wallet can be accessed anywhere, why add extra?

Reply:
#235 prevents accessing the selected wallet from anywhere. That approach is buggy, this is better.